.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,509 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="OTHER_INDENT_OPTIONS">
+      <value>
+        <option name="INDENT_SIZE" value="2" />
+        <option name="CONTINUATION_INDENT_SIZE" value="4" />
+        <option name="TAB_SIZE" value="8" />
+      </value>
+    </option>
+    <option name="LINE_SEPARATOR" value="&#10;" />
+    <option name="RIGHT_MARGIN" value="140" />
+    <option name="FORMATTER_TAGS_ENABLED" value="true" />
+    <option name="LINE_COMMENT_AT_FIRST_COLUMN" value="false" />
+    <option name="BLOCK_COMMENT_AT_FIRST_COLUMN" value="false" />
+    <GroovyCodeStyleSettings>
+      <option name="STATIC_METHODS_ORDER_WEIGHT" value="5" />
+      <option name="METHODS_ORDER_WEIGHT" value="4" />
+    </GroovyCodeStyleSettings>
+    <JavaCodeStyleSettings>
+      <option name="PREFER_LONGER_NAMES" value="false" />
+      <option name="GENERATE_FINAL_LOCALS" value="true" />
```

Comment:
I'd prefer not to have this. Currently, my code generation settings (from Android Studio) look like this:

<img width="308" alt="screen shot 2018-12-10 at 4 41 49 pm" src="https://user-images.githubusercontent.com/8518285/49770746-caa87180-fc9a-11e8-9fea-95e602d4e907.png">

Highlighting can be used to easily identify variables that get reused. I see them underlined, which is enough to get me to use a fresh var when it makes sense.

<img width="1122" alt="screen shot 2018-12-10 at 4 36 34 pm" src="https://user-images.githubusercontent.com/8518285/49770758-d5630680-fc9a-11e8-9a4f-f70412aaea8f.png">

Reply:
Works for me.